continuous_integration/environment-mindeps-array-dataframe.yaml

```diff
@@ -11,8 +11,8 @@ dependencies:
   - fsspec=0.6.0
   - toolz=0.8.2
   # optional dependencies pulled in by pip install dask[array, dataframe]
-  - numpy=1.16
-  - pandas=0.25.0
+  - numpy=1.18
+  - pandas=1.0.0
```

Comment:
Yeah I thought users should be upgrading to the latest patch release since that should be non-breaking. Plus this would save us working around bugs that have been fixed by upstream

That all being said, don't have strong feelings here if we want to do something different :)